### PR TITLE
portfolio mobile set up

### PR DESCRIPTION
**Pull request by glorymugizi**
Hi,
Added linters.
Added html file.
Styled the nav bar and heading section in the header element.
updated the REAME.md

Thank you for reviewing my pull request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  <body>
+    <header>
+      <nav>
+        <ul class="tool-bar">
+          <li class="my-logo">MY LOGO</li>
+          <li class="toggle-menu">
+            <img src="images/humbger.png" alt="" srcset="" />
+          </li>
+        </ul>
+      </nav>
```
Comment (ha-manel) [inline index.html]: - [ ] Kindly wrap only the navbar in a `header` tag and not the whole headline section for better semantics 👍 
- [ ] Kindly remove the `ul` element and the `li` elements, they're not a list so it's better for semantics if we don't use the `ul` here.
- [ ] Kindly wrap the logo in an `a` tag with `href="#"`, this way the logo will take to the top of the page when we click on it 💯

File: index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          <h1>I’m Glory Mugizi Glad to see you!</h1>
+        </div>
+        <div class="container paragraph-one">
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <a href="" class="lets-connect">LET’S CONNECT</a>
```
Comment (ha-manel) [inline index.html]: - [ ] This isn't a link so I suggest you use a text tag like `p` instead of `a` tag 👍

File: index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+          <ul class="social-media-links">
+            <li>
+              <a href="#"><img src="images/twitter.png" alt="twitter"/></a>
+            </li>
```
Comment (ha-manel) [inline index.html]: Awesome job on using the right tags here 👏

File: index.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        <ul class="tool-bar">
+          <li class="my-logo">MY LOGO</li>
+          <li class="toggle-menu">
+            <img src="images/humbger.png" alt="" srcset="" />
+          </li>
+        </ul>
+      </nav>
+      <section class="headline">
+        <div class="container heading">
+          <h1>I’m Glory Mugizi Glad to see you!</h1>
```
Comment (ha-manel) [inline index.html]: - [ ] Great job on using the `h1` tag correctly 💯 Kindly use the right `font-family` and the right text `color` to match the design in Figma 👍 (this applies to all the texts in the page}

## Your design:
image

## Figma design:
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        </div>
+        <div class="container paragraph-one">
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <a href="" class="lets-connect">LET’S CONNECT</a>
+          <ul class="social-media-links">
```
Comment (ha-manel) [inline index.html]: - [ ] Kindly reduce the spaces between the social media icons to match the design in Figma 💯 

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @glorymugizi 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Professional README file ✔️

## Required Changes:
- [ ] From now on, kindly choose the branch's name more carefully, it should be an action describing what will be done in the branch, example: "create-portfolio-headline" 👍 
- [ ] Kindly try to write better commit messages that follow best practices, they should be capitalized sentences in the imperative form, example: "Add linters" instead of "added linters"

_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (glorymugizi) [reply index.html]: Hell @ha-manel,
The navbar was wrapped, ul element removed and logo wrapped in a with href="#".
Thank you.

File: index.html (hunk 2 above)
Comment (glorymugizi) [reply index.html]: Hi @ha-manel,
a tag was replaced with p tag as requested.
Thank you.

File: index.html (hunk 3 above)
Comment (glorymugizi) [reply index.html]: @ha-manel.
Thank you.

File: index.html (hunk 5 above)
Comment (glorymugizi) [reply index.html]: Hi @ha-manel,
spaced reduced as suggested.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+  <body>
+    <header>
+      <nav>
+        <div class="navbar">
+          <div class="my-logo"><a href="#">My Logo</a></div>
+          <div class="toggle-menu">
+            <img src="images/humbger.png" alt="toggle menu" />
+          </div>
+        </div>
+      </nav></header>
```
Comment (iambenkis) [inline index.html]: - [ ] [**OPTIONAL**] Kindly make sure the ```font-size``` of your design is the as use in the Figma design.
```bash
    .my-logo {
font-size:  18px
}
```

Your design:

Screenshot from 2022-12-02 10-47-00


The Figma design:

Screenshot from 2022-12-02 10-53-40

File: index.html [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+      <section class="headline">
+        <div class="container heading">
+          <h1>I’m Glory Mugizi Glad to see you!</h1>
```
Comment (iambenkis) [inline index.html]: - [ ] As the previous code reviewer was explaining , **always make sure to stick as much as possible to the Figma design. You should use the Figma design as a reference for your ```colors``` , ```font-size```, ```margins```, ```paddings```, ```font-weight```, and so forth** :100:. You did a great job using the same ```colour``` and ```font-size``` :+1:  however, the ```font-weight``` you use here should be the same as in the Figma design. 

Your design : 

Screenshot from 2022-12-02 10-46-44

The Figma design:

Screenshot from 2022-12-02 10-53-55

**Review by iambenkis — CHANGES_REQUESTED** [review]
Hi @glorymugizi  , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question, so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (glorymugizi) [reply index.html]: Hi @iambenkis, the figma design has 18px as font-size when I implement it the way it is, it's too small from the design. Please advise on how to move forward. You may please have a look at my CSS and check it out.

File: index.html (hunk 7 above)
Comment (glorymugizi) [reply index.html]: Hi @iambenkis, the Figma design has 40px as font size when I implement it the way it is, it's too small from the design. Please advise on how to move forward. You may please have a look at my CSS  too.
if I set the font-weight as per the Figma design it appears to differ from it.

Comment (iambenkis) [reply index.html]: @glorymugizi, Kindly stick to the Figma design

Comment (iambenkis) [reply index.html]: Make sure the font-weight is the same

File: index.html (hunk 6 above)
Comment (glorymugizi) [reply index.html]: Hi @iambenkis,
Changed the logo font size to 18px  and the heading as per your requested suggestions.

Thank you.

File: index.html (hunk 7 above)
Comment (glorymugizi) [reply index.html]: Hi @iambenkis,
Changed the logo font size to 18px  and the heading as per your requested suggestions.

Thank you.

**Comment by iambenkis** [comment]
Mostly welcome 🙏 
@glorymugizi now you can request for another code review

File: index.html (hunk 6 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 7 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @glorymugizi 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**